File: AtCoder/AtCoder Beginner Contest/081/B/Go/ABC_081_B_01.go
package main

import "fmt"

func main() {
	var N int
	fmt.Scanf("%d", &N)
	A := make([]int, N)
	count := 0
	flg := 0
	for i := 0; i < N; i++ {
		fmt.Scanf("%d", &A[i])
	}

	for flg == 0 {
		for i := 0; i < N; i++ {
			if A[i] % 2 != 0 {
				flg = 1
				break
			}
			A[i] = A[i] / 2
		}
		count++
	}
	fmt.Printf("%d\n", count-1)
}

File: AtCoder/AtCoder Beginner Contest/081/B/C/ABC_081_B_01.c
#include <stdio.h>

int main(void)
{
    int N;
    int A[N];
    int i, flg = 0, count = 0;

    scanf("%d", &N);
    for (i = 0; i < N; i++) {
        scanf("%d", &A[i]);
    }

    while (flg == 0) {
        for (i = 0; i < N; i++) {
            if (A[i] % 2 != 0) {
                flg = 1;
                break;
            }
            A[i] = A[i] / 2;
        }
        count++;
    }
    printf("%d\n", count-1);
    return 0;
}